rkt/enter.go

```diff
@@ -57,27 +57,21 @@ func runEnter(cmd *cobra.Command, args []string) (exit int) {
 		return 1
 	}
 
-	podUUID, err := resolveUUID(args[0])
+	p, err := getPodFromUUIDString(args[0])
 	if err != nil {
-		stderr("Unable to resolve UUID: %v", err)
-		return 1
-	}
-
-	p, err := getPod(podUUID)
-	if err != nil {
-		stderr("Failed to open pod %q: %v", podUUID, err)
+		stderr("%v", err)
```

Comment:
If you print the error direcly from the function it will be lower-cased, can you prefix it with some meaningful text?